doc-content/jbpm-docs/src/main/asciidoc/Persistence/persistence-dependencies-con.adoc

```diff
@@ -0,0 +1,48 @@
+[id='persistence-dependencies-con_{context}']
+= Providing dependencies for persistence
+
+To use persistence, you mustmake sure the necessary dependencies are available in the class path of your application
+
+By default, persistence is based on the Java Persistence API (JPA) and can work with several persistence mechanisms. By default, Hibernate is used.
+
+If youuse  the Eclipse IDE and the jBPM Eclipse plugin, make sure the necessary JARs are added to your jBPM runtime directory. If you are using the runtime that is configured by default by the {PRODUCT} installer, the JARs are normally already available.
+
+You can also download the
+ifdef::JBPM,DROOLS,OP[]
+jBPM runtime artifact from the jBPM downloads
+endif::JBPM,DROOLS,OP[]
+ifdef::PAM,DM[]
+`{PRODUCT_FILE}-maven-repository.zip` product deliverable file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal
+endif::PAM,DM[]
+and configure the jBPM plugin to use the artifacts from this file.
+
+To add the necessary dependencies to your project manually, you must place the `jbpm-persistence-jpa.jar` file in your project class path. This file contains the code for saving the runtime state whenever necessary. Depending on the persistence solution and database you are using, you might need additional dependencies.
+
+For the default combination of:
```

Comment:
This is an interesting case for the IBM SG. Making each of these list lead-ins a complete sentence would be tricky. So I'd at least make this a "complete fragment" as it were :) , such as "For the default combination of the following persistence components:" Just a thought.

Reply:
Gone anyway :)